src/test/java/org/stellar/sdk/requests/AccountsRequestBuilderTest.java

```diff
@@ -18,4 +19,23 @@ public void testAccounts() {
             .buildUri();
     assertEquals("https://horizon-testnet.stellar.org/accounts?cursor=13537736921089&limit=200&order=asc", uri.toString());
   }
+
+  @Test
+  public void testForSigner() {
+    Server server = new Server("https://horizon-testnet.stellar.org");
+    HttpUrl uri = server.accounts()
+            .forSigner("GDSBCQO34HWPGUGQSP3QBFEXVTSR2PW46UIGTHVWGWJGQKH3AFNHXHXN")
+            .buildUri();
+    assertEquals("https://horizon-testnet.stellar.org/accounts?signer=GDSBCQO34HWPGUGQSP3QBFEXVTSR2PW46UIGTHVWGWJGQKH3AFNHXHXN", uri.toString());
+  }
+
+  @Test
+  public void testForAsset() {
```

Comment:
could you add a test case where the asset is native and another test case where the asset has type AssetTypeCreditAlphaNum12?

Reply:
This parameter should not be a native asset, so I changed the parameter type to `AssetTypeCreditAlphaNum` and added some tests.